meshtastic/mesh.proto

```diff
@@ -636,6 +636,8 @@ enum HardwareModel {
   /* M5 esp32 based MCU modules with enclosure, TFT and LORA Shields. All Variants (Basic, Core, Fire, Core2, CoreS3, Paper) https://m5stack.com/ */
   M5STACK_CORES3=80;
   
+  /* Seeed XIAO S3 DK
+  M5STACK_CORES3=81;
```

Comment:
Looks like you may be missing a close comment (*/) here.

Reply:
plus i am pretty sure it should not be named "M5STACK_CORES3" ;-)